axum/src/response/sse.rs

```diff
@@ -174,6 +176,27 @@ pub struct Event {
     flags: EventFlags,
 }
 
+#[derive(Debug)]
+#[must_use]
+/// Expose [`Event`] as a [`std::fmt::Write`]
```

Comment:
This is pretty unusual, I don't think we put regular attributes above doc comments anywhere else. Can you swap it?